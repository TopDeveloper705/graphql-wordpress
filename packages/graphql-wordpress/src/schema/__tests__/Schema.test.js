import Schema from '../../schema';

describe('Test Schema', () => {
  test('Types', () => {
    expect(Schema.getTypeMap()).toMatchSnapshot();
  });
});
