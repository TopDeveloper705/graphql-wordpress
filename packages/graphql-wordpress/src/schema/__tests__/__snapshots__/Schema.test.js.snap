// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Schema Types 1`] = `
Object {
  "AddCommentInput": "AddCommentInput",
  "AddCommentPayload": "AddCommentPayload",
  "Audio": "Audio",
  "AudioDetails": "AudioDetails",
  "Avatar": "Avatar",
  "Boolean": "Boolean",
  "COMMENT_ORDERBY": "COMMENT_ORDERBY",
  "COMMENT_STATUS": "COMMENT_STATUS",
  "Caption": "Caption",
  "Category": "Category",
  "Chart": "Chart",
  "ChartItem": "ChartItem",
  "ChartItemImage": "ChartItemImage",
  "Comment": "Comment",
  "CommentConnection": "CommentConnection",
  "CommentEdge": "CommentEdge",
  "Content": "Content",
  "ContentNode": "ContentNode",
  "DeleteCommentInput": "DeleteCommentInput",
  "DeleteCommentPayload": "DeleteCommentPayload",
  "Description": "Description",
  "Element": "Element",
  "Embed": "Embed",
  "Excerpt": "Excerpt",
  "FORMAT": "FORMAT",
  "Guid": "Guid",
  "ID": "ID",
  "Image": "Image",
  "ImageDetails": "ImageDetails",
  "Int": "Int",
  "Labels": "Labels",
  "MEDIA_TYPE": "MEDIA_TYPE",
  "Media": "Media",
  "MediaInterface": "MediaInterface",
  "MediaSize": "MediaSize",
  "Meta": "Meta",
  "Mutation": "Mutation",
  "NavMenu": "NavMenu",
  "NavMenuItem": "NavMenuItem",
  "Node": "Node",
  "ORDER": "ORDER",
  "PING_STATUS": "PING_STATUS",
  "POST_ORDERBY": "POST_ORDERBY",
  "Page": "Page",
  "PageInfo": "PageInfo",
  "Post": "Post",
  "PostConnection": "PostConnection",
  "PostEdge": "PostEdge",
  "PostInterface": "PostInterface",
  "Query": "Query",
  "Rewrite": "Rewrite",
  "Settings": "Settings",
  "Sidebar": "Sidebar",
  "String": "String",
  "Tag": "Tag",
  "Taxonomy": "Taxonomy",
  "TermInterface": "TermInterface",
  "Text": "Text",
  "Title": "Title",
  "UpdateCommentInput": "UpdateCommentInput",
  "UpdateCommentPayload": "UpdateCommentPayload",
  "User": "User",
  "Video": "Video",
  "VideoDetails": "VideoDetails",
  "Viewer": "Viewer",
  "Widget": "Widget",
  "__Directive": "__Directive",
  "__DirectiveLocation": "__DirectiveLocation",
  "__EnumValue": "__EnumValue",
  "__Field": "__Field",
  "__InputValue": "__InputValue",
  "__Schema": "__Schema",
  "__Type": "__Type",
  "__TypeKind": "__TypeKind",
}
`;
